### PR TITLE
Updated deployment template to use string substitution when applicable

### PR DESCRIPTION
**Pull request by dymurray**
**Describe what this PR does and why we need it**:
This PR fixes yaml errors when using characters in a password
Changes proposed in this pull request
 - Use strings for config map

### CODE REVIEW

File: templates/deploy-ansible-service-broker.template.yaml [hunk 1]
```diff
@@ -177,14 +177,14 @@ objects:
         level: debug
         color: true
       openshift:
-        host: ${CLUSTER_AUTH_HOST}
-        ca_file: ${CA_FILE}
-        bearer_token_file: ${BEARER_TOKEN_FILE}
-        image_pull_policy: ${IMAGE_PULL_POLICY}
+        host: "${CLUSTER_AUTH_HOST}"
+        ca_file: "{CA_FILE}"
```
Comment (jmrodri) [inline templates/deploy-ansible-service-broker.template.yaml]: You removed the $ from {CA_FILE} this causes the ca_file setting to be "{CA_FILE}" which is not correct.

**Comment by jmrodri** [comment]
CI failed

File: templates/deploy-ansible-service-broker.template.yaml [hunk 2]
```diff
@@ -177,14 +177,14 @@ objects:
...
+        host: "${CLUSTER_AUTH_HOST}"
+        ca_file: "${CA_FILE}"
+        bearer_token_file: "${BEARER_TOKEN_FILE}"
+        image_pull_policy: "${IMAGE_PULL_POLICY}"
       broker:
         dev_broker: ${DEV_BROKER}
         bootstrap_on_startup: ${BOOTSTRAP_ON_STARTUP}
-        refresh_interval: ${REFRESH_INTERVAL}
+        refresh_interval: "${REFRESH_INTERVAL}"
         launch_apb_on_bind: ${LAUNCH_APB_ON_BIND}
```
Comment (rthallisey) [inline templates/deploy-ansible-service-broker.template.yaml]: Any reason why we shouldn't quotes all the variables in the configmap?

Comment (dymurray) [inline templates/deploy-ansible-service-broker.template.yaml]: Yes booleans fail if you convert them into strings when umarshalling the yaml file. So everything is treated as a string other than bools.

**Pull request merged**